src/main/java/com/JayPi4c/NeuroEvolution/controller/LanguageController.java

```diff
@@ -33,6 +32,7 @@ public LanguageController(MainStage stage) {
 		stage.getDrawer().getTrackCard().getHeader().textProperty().bind(createBinding("drawer.card.track.header"));
 		stage.getDrawer().getTrackCard().getPerlinTrackLabel().textProperty().bind(createBinding("drawer.card.track.perlin"));
 		stage.getDrawer().getTrackCard().getConvexHullLabel().textProperty().bind(createBinding("drawer.card.track.convexHull"));
+        stage.getDrawer().getTrackCard().getPartTrackLabel().textProperty().bind(createBinding("drawer.card.track.newTrack"));
```

Comment:
In den messages Dateien ist der identifier "drawer.card.track.partTrack" genannt. Hier wird aber der Identifier "....newTrack" angesprochen
```suggestion
stage.getDrawer().getTrackCard().getPartTrackLabel().textProperty().bind(createBinding("drawer.card.track.partTrack"));
```